### DISCUSSION TITLE
handwritten

### DISCUSSION THREAD

**Discussion by dhea1323** [mindee/doctr#834] [Q&A]
Hello,
Is it possible for docTR to collaborate with a text recognition model for handwritten pages?. Has anyone tried it and want to share about this?

**Comment by fg-mindee** [comment]
Hello @dhea1323 :wave: 

Thanks for asking! We're very open about external contributions, and we're in the middle of having an easier way for contributors to share their models. Just to be clear, do you already have a model for handwritten recognition? Or are you looking for one?

We don't have a trained model for handwritten recognition yet if that's what you are looking for :upside_down_face:
